fhir-server-test/src/test/java/com/ibm/fhir/server/test/BundleTest.java

```diff
@@ -156,7 +156,9 @@ public Bundle getEntityWithExtraWork(Response response, String method) throws Ex
     public void commonWork(Bundle responseBundle, String method) throws Exception{
         assertNotNull(responseBundle);
         printBundle(method, "response", responseBundle);
-        checkForIssuesWithValidation(responseBundle, true, false);
+        if(DEBUG) {
+            checkForIssuesWithValidation(responseBundle, true, false);
+        }
```

Comment:
what are we missing by skipping this step by default?

Reply:
in this case, it's just outputing the bundle report and was driving a lot of logging.  
Mainly missing narrative statements.

Reply:
Clarified (added a comment)

Reply:
refactored slightly... pushed the debugging to checkForIssuesWithValidation (I pass DEBUG down to it, so we can see/not see non-fatal/warning errors. )